Modules/Quotas/src/Quotas.Application/Identities/Queries/ListQuotasForIdentity/ListQuotasForIdentityResponse.cs

```diff
@@ -11,14 +11,14 @@ public class ListQuotasForIdentityResponse : CollectionResponseBase<QuotaGroupDT
 public class SingleQuotaDTO
 {
     public QuotaSource Source { get; set; }
-    public string MetricKey { get; set; }
+    public required string MetricKey { get; set; }
     public int Max { get; set; }
     public uint Usage { get; set; }
-    public string Period { get; set; }
+    public required string Period { get; set; }
 }
```

Comment:
Judging by the instancing of this class through initializers, it seems that all five properties (lines 13-17) can be set as `required`. Even though the current situation doesn't produce any warnings, I think this would be a good thing to do.

@tnotheis what do you think?

Reply:
👍